src/main/java/com/stuypulse/robot/subsystems/Shooter.java

```diff
@@ -79,28 +81,34 @@ public Shooter() {
 
         shooterPIDController = shooterMotor.getPIDController();
         feederPIDController = feederMotor.getPIDController();
+        configure();
 
-        shooterPIDController.setP(ShooterSettings.ShooterPID.kP);
-        shooterPIDController.setI(ShooterSettings.ShooterPID.kI);
-        shooterPIDController.setD(ShooterSettings.ShooterPID.kD);
-        shooterPIDController.setFF(ShooterSettings.ShooterPID.kF);
+        Motors.Shooter.LEFT.configure(shooterMotor);
+        Motors.Shooter.RIGHT.configure(shooterFollower);
+        Motors.Shooter.FEEDER.configure(feederMotor);
 
-        feederPIDController.setP(ShooterSettings.FeederPID.kP);
-        feederPIDController.setI(ShooterSettings.FeederPID.kI);
-        feederPIDController.setD(ShooterSettings.FeederPID.kD);
-        feederPIDController.setFF(ShooterSettings.FeederPID.kF);
+        // Setup Solenoid
+        hoodSolenoid = new Solenoid(PneumaticsModuleType.CTREPCM, Ports.Shooter.HOOD_SOLENOID);
+    }
 
-        MotorSettings.Shooter.LEFT.configure(shooterMotor);
-        MotorSettings.Shooter.RIGHT.configure(shooterFollower);
-        MotorSettings.Shooter.FEEDER.configure(feederMotor);
+    private void configure() {
```

Comment:
This could probably be renamed to make it clear that it is configuring constants